extensions/data-transfer/portability-data-transfer-daybook/src/main/java/org/datatransferproject/transfer/daybook/photos/DaybookPhotosImporter.java

```diff
@@ -127,7 +128,7 @@ private int importPhoto(
       inputStream = new URL(photoModel.getFetchableUrl()).openStream();
     } else {
       monitor.severe(() -> "Can't get inputStream for a photo");
```

Comment:
Big improvement in-passing here, thakns for doing this; (`IOException` > `-1` in my book)


just a nitpick below to factor out the message and document the magic `null` being passed.
```suggestion
      String errorMessage = "Can't get inputStream for a photo";
      monitor.severe(() -> errorMessage);
      return ItemImportResult.error(new IOException(errorMessage), null /*sizeInBytes*/);
```

Reply:
I generally don't do argument comments because IDEA does exactly that automatically
<img width="540" alt="Screenshot 2022-07-04 at 11 54 50 am" src="https://user-images.githubusercontent.com/426393/177140835-f097a8e0-2caf-4e7b-80e2-46fca18920e4.png">


I don't mind adding comments as well, but I'd prefer it to be a universal rule all of us can always stick to. Shall we codify a rule? I wonder if it is doable through google-java-format config.

Reply:
tl;dr I've added to our ongoing "things to discuss" doc.

> I generally don't do argument comments because IDEA does exactly that automatically

Ah, yeah makes sense. Doesn't work for someone without that IDE feature, but I think to your point below - it's more important that we're consistent than that we have to bother figuring things out like this on every review.

> I don't mind adding comments as well, but I'd prefer it to be a universal rule all of us can always stick to. Shall we codify a rule? I wonder if it is doable through google-java-format config.

100% agree; sorry I'm used to a particular style guide and forget I'm not working with that in this project :facepalm: 
But yeah, going forward, we should just let something enforce .. _some_ styleguide for us so we don't even have to discuss.

Reply:
Oh, whoops - unmarking "resolved" as I think your thumbs-up was that you still wanted to factor out the errorMessage.

(feel free to mark resolved to merge - I don't think this change is a big deal!)